test/FunctionalTests/Tests/DataOData/Tests/OData.Scenario.Tests/UriParser/SelectExpand/SelectExpandTest.DeepSelectionsAreRedundantIfEntireSubtreeIsSelected.approved.txt

```diff
@@ -22,5 +22,4 @@ SelectExpandQueryOption
 										SelectedItems
 											Path]
```

Comment:
Is this a bug in the test or the code?  I would have expected this to write out the selected function.

Reply:
The selected function "OrdersWithMoreThanTwoItems" is bound to entity collection but not the entity type.

---
In reply to: [224206728](https://github.com/OData/odata.net/pull/1286#discussion_r224206728) [](ancestors = 224206728)